src/main/resources/assets/js/app/highchart.es6

```diff
@@ -100,6 +94,98 @@ export function init() {
             $(e.browserEvent.target).toggleClass('disabled')
           }
         }
+
+        const category = 'Highcharts'
+        const action = 'Lastet ned highcharts'
+
+        config.exporting.menuItemDefinitions = {
```

Comment:
Kul måte å legge til knappene på! Den skal jeg kanskje låne. 
Men: Gjør den at disse blir utdatert eller trenger vi da begge? 
https://github.com/statisticsnorway/mimir/blob/e5fb7e0a6d49e3d89c8a2c8bd9b7aac4bf1baf9d/src/main/resources/lib/ssb/parts/highcharts/graph/config.es6#L103

Reply:
Vi trenger begge! I menuItems lister vi de valgene vi vil ha med, for uten så listes alle mulige valg og disse bruker sin default konfig dvs text, onclick osv. Hvis vi vil overskrive konfigen så gjøres det i menuItemDefinitions. menuItemDefinitions kan egentlig settes i config.es6, men i dette tilfellet så funket det bare med jquery :(